### PR TITLE
CW zip with

### PR DESCRIPTION
**Pull request by aepaysinger**

### CODE REVIEW

File: code_challenges/code_wars/zip_with.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class ZipWith:
+    def __init__(self, func, array1, array2):
+        self.func = func
+        self.array1 = array1
+        self.array2 = array2
+
+    def zip_with(self):
+        return [self.func(a, b) for a, b in zip(self.array1, self.array2)]
```
Comment (W-Ely) [inline code_challenges/code_wars/zip_with.py:11]: Part of the purpose of classes is reusability.  This class should be instantiated with a zip function, but be able to zip any 2 arrays.

File: code_challenges/code_wars/zip_with.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from operator import add, sub, mul
```
Comment (W-Ely) [inline code_challenges/code_wars/zip_with.py:1]: unused imports should be removed.

**Review by W-Ely — CHANGES_REQUESTED** [review]

File: code_challenges/code_wars/zip_with.py (hunk 2 above)
Comment (aepaysinger) [reply code_challenges/code_wars/zip_with.py:1]: i have that there for my __if__ name == "__main__" block.

Comment (W-Ely) [reply code_challenges/code_wars/zip_with.py:1]: Are all three used in your if name == main block?  If not, remove the ones that are not used.  Unused imports should not be present.

**Review by W-Ely — APPROVED** [review]

**Review by W-Ely — APPROVED** [review]

**Pull request merged**